modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -10,7 +10,7 @@ to estimate the number of pods that can be scheduled onto your cluster.
 
 .Prerequisites
 
-* Download and install link:https://github.com/kubernetes-incubator/cluster-capacity[the *cluster-capacity* tool].
+* Download and install link:https://catalog.redhat.com/software/containers/openshift4/ose-cluster-capacity/5cca0324d70cc57c44ae8eb6?container-tabs=overview[the *cluster-capacity* tool].
```

Comment:
As it's now run as a container image through Podman, should this be reworded as we pull and run the image rather than actually installing?

Reply:
Thanks, Martin. I'v made that change. @martineg